frontend/src/css/password.css

```diff
@@ -0,0 +1,83 @@
+#logo {
+  float: left;
+  padding-top: 20px;
+  padding-left: 20px;
+}
+
+#text {
+  color: #054D74;
+  width: 575px;
+  height: 265px;
+  /* background-color: brown; */
+  
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  
+  margin: auto;
+  text-align: left;
+}
+
+#forgotPassword {
+  margin-top: 0px;
+  float: left;
+  font-size: 57.7px;
+}
+
+#enterEmailText {
+  color: black;
+  float: left;
+  font-size: 27px;
+}
+
+#enterYourEmail {
+  margin-top: 25px;
+  width: 569px;
+  height: 30px;
+  border-radius: 5px;
+  border-width: 1px;
+  font-size: 18px;
+  padding-left: 15px;
+}
+
+#send {
+  margin-top: 15px;
+  width: 588px;
+  height: 45px;
+  background-color: #4AA7AC;
+  color: white;
+  border-radius: 5px;
+  border-width: 0px;
+  font-size: 18px;
+}
+
+#inputFields {
+  float: top;
+}
+
+#toLogin {
+  float: right;
+}
+
+#resendEmail {
+  font-size: 18px;
+}
+
+#enterYourPassword {
+  margin-top: 25px;
+  width: 569px;
+  height: 30px;
+  border-radius: 5px;
+  border-width: 1px;
+  font-size: 18px;
+  padding-left: 15px;
+}
+
+#resetYourPassword {
```

Comment:
Same comments as above. Smaller width, user Material-UI textfield components to take in input